src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -443,6 +417,23 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
       };
     };
 
+    async function loadSatelliteAssemblies(culturesToLoad: string[], loader: (wrapper: { dll: Uint8Array }) => void): Promise<void> {
+      const satelliteResources = resourceLoader.bootConfig.resources.satelliteResources;
+      if (!satelliteResources) {
+        return;
+      }
+      await Promise.all(culturesToLoad!
+        .filter(culture => satelliteResources.hasOwnProperty(culture))
+        .map(culture => resourceLoader.loadResources(satelliteResources[culture], fileName => `_framework/${fileName}`, 'assembly'))
+        .reduce((previous, next) => previous.concat(next), new Array<LoadingResource>())
```

Comment:
I know this line was already present in the earlier version of the code, but isn't it a no-op? It seems to walk through an array and concatenate all its entries into an array (i.e., producing the same array as it started with). Is the code the same if this line is removed?

Reply:
this is adding list of resouces from multiple cultures, I think it needs to be there.

Reply:
Oh, so each `loadResources` call returns an array with multiple values? If so makes sense!